packages/botonic-react/tests/components/multichannel/__snapshots__/multichannel-carousel.test.jsx.snap

```diff
@@ -69,31 +69,17 @@ exports[`Multichannel carousel COMPACT mode dynamic carousel with URL and postba
     </button>
   </message>,
   <message
-    buttonsAsText={0}
-    delay={0}
-    indexMode="number"
-    markdown={0}
-    newkey={2}
-    type="text"
-    typing={0}
-  >
-    *Snatch* _Five minutes, Turkish_
-    
-- Visit website: https://www.imdb.com/title/tt0208092
-  </message>,
+    body="_Snatch_ _Five minutes, Turkish_"
+    displayText="Visit website"
+    type="whatsapp-cta-url-button"
+    url="https://www.imdb.com/title/tt0208092"
+  />,
```

Comment:
why carousel snapshots are changing?

Reply:
these are the ones when Multichannel is used. All texts with only one button will be converted to WhatsappCTAUrlButton in Whatsapp